README.md

```diff
@@ -0,0 +1,27 @@
+# AI-IaC-Prompts
```

Comment:
I would recommend also explaining the goals of this repo in a bit more detail:
- What's the reason we've created this repo?
- Who is this for?
- How do we plan to use the results?
- Mention that we're planning to use upvotes to guide prioritization.
- Explanation of what "Deployments" means (e.g. bicep + terraform / other IaC)

Might also be worth creating some example issues - to give an idea of the sort of input we're looking for.